pkgs/development/python-modules/pyscard/default.nix

```diff
@@ -9,24 +17,28 @@ buildPythonPackage rec {
     sha256 = "15fh00z1an6r5j7hrz3jlq0rb3jygwf3x4jcwsa008bv8vpcg7gm";
   };
 
-  postPatch = ''
-    sed -e 's!"libpcsclite\.so\.1"!"${stdenv.lib.getLib pcsclite}/lib/libpcsclite.so.1"!' \
-        -i smartcard/scard/winscarddll.c
+  postPatch = if withApplePCSC then ''
+    substituteInPlace smartcard/scard/winscarddll.c \
+      --replace "/System/Library/Frameworks/PCSC.framework/PCSC" \
+                "${PCSC}/Library/Frameworks/PCSC.framework/PCSC"
```

Comment:
I'm not familiar with the details of Apple SDK wrappers. This latter appears to be symlink to the former, is there any point to this?

Reply:
If we ever get sandboxing enabled by default, it will be necessary to have the latter to ensure that Nix includes the correct framework.